api/config/settings.py

```diff
@@ -312,9 +313,9 @@
 MOBILE_ACTIVATION_TOKEN_HASH_ALGORITHM = 'sha256'
 
 # Test Settings
```

Comment:
Commented code should not be pushed @Ogutu-Brian. Kindly change that.

Reply:
Hello @Bakley ,
That is a comment on what is actually going on in the code,
It is not commented code.